JoinColony/colonyCDapp
# test: add e2e tests for 1st step of create colony flow

## Description

I've broken down the initial PR into smaller parts to help identify the cause of the inconsistent test results. This is the first part covering the 1st step of the Create colony form. 

- Added e2e tests for the Create Colony flow / Details step
- Set up playwright eslint plugin


## Testing

**Prerequisites**: The test script simulates user interactions with the local instance of the web app, which means that the BE is expected to be set up and running. For now, backend services should be started manually and the initial test data should be seeded . Therefore, ensure that the development environment is properly set up by following the instructions in the README file

- Start your local dev environment  `npm run dev` 
- Seed the mock data `node scripts/create-data.js` 
- If it's the first time you run e2e tests on your machine - run `npm e2e:install` 

- run `npm run e2e` or `npm run e2e:ui`
- Ensure all tests pass

## Diffs

**New stuff** ✨

- playwright folder is added to the lint script
- added loading of the .env file into playwright environment
